common-arcana.lic

```diff
@@ -49,6 +49,9 @@ module DRCA
     when 'Your desire to prepare this offensive spell suddenly slips away'
       pause 1
       return prepare?(abbrev, mana, symbiosis, command)
+    when 'You should stop playing before you do that'
+      fput('stop play')
+      return prepare?(abbrev, mana, symbiosis, command)
```

Comment:
This could result in preparing symbiosis twice, but it looks like that case is handled. Not elegant, but not a problem (I think).